src/widgets/allergies/allergy-intolerance.resource.tsx

```diff
@@ -31,13 +31,13 @@ export function fetchAllergyByUuid(allergyUuid: string) {
 function mapAllergyProperties(allergy) {
```

Comment:
Missing type (sorry I know this is not your change, but otherwise its hard to reason about the rest)

Reply:
Updated.